modules/ephemeral-storage-csi-inline-overview.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/ephemeral-storage-csi-inline.adoc
+
+[id="ephemeral-storage-csi-inline-overview_{context}"]
+= Overview of CSI inline volumes
```

Comment:
From the content, suggest the title is `Overview of CSI inline ephemeral volumes`

Reply:
I debated using that wording, but chose to stay with [PM's title](https://issues.redhat.com/browse/STOR-297) for consistency.